app/src/main/java/io/appium/uiautomator2/handler/GetElementAttribute.java

```diff
@@ -102,6 +84,25 @@ public AppiumResponse safeHandle(IHttpRequest request) {
 
     }
 
+    public static String getElementAttributeValue(AndroidElement element, String attributeName) throws NoAttributeFoundException, UiObjectNotFoundException, ReflectiveOperationException {
+        if ("name".equals(attributeName) || "contentDescription".equals(attributeName)
```

Comment:
can we rewrite this as `Arrays.asList(....).contains(attributeName)` ?

Reply:
Fine by me; I just copied it as-is from the original implementation